examples/compositor/wl_compositor.c

```diff
@@ -8,7 +8,7 @@
 
 static void destroy_surface_listener(struct wl_listener *listener, void *data) {
 	struct wlr_surface *surface = wl_resource_get_user_data(data);
-	struct wl_compositor_state *state = surface->compositor_data;
```

Comment:
Should I create both a `compositor_data` and `data` field in wlr_surface then? I think it's harmless until wl_compositor gets merged into wlroots. Also I created #71 for this